DNN Platform/Website/development.config

```diff
@@ -360,10 +360,11 @@
         <add name="DNNProfileProvider" type="DotNetNuke.Security.Profile.DNNProfileProvider, DotNetNuke" providerPath="~\Providers\MembershipProviders\DNNProfileProvider\"/>
       </providers>
     </profiles>
-    <permissions defaultProvider="CorePermissionProvider">
+    <permissions defaultProvider="DetailedPermissionProvider">
```

Comment:
Do we want this to be the new standard?  I think it should be but @dnnsoftware/approvers thoughts?

Reply:
I'm not an approver ;)
...but I really think it should be.

Note that @tvatavuk will probably push another update within a few days, with a `CmsPermissionProvider` or similar, which would then make Editors / Content Managers also become part of the new standard. Not sure yet if we should have 2 providers - so people could choose with/without Editors/Managers, but if we have both, then the question of the default should be asked in that context.

Reply:
I think it should be the new standard.

Reply:
Agreed